src/types.ts

```diff
@@ -449,6 +452,16 @@ export interface LeaderboardPnlResponseObject {
   startedAt: ISO8601 | null;
   endsAt: ISO8601 | null;
   updatedAt: ISO8601;
+
+  // leagues specific
```

Comment:
remove this and put a comment at the top that some values are league specific or something. We've never done this style